app/userland/settings/js/views/general.js

```diff
@@ -484,6 +545,35 @@ class GeneralSettingsView extends LitElement {
     this.requestUpdate()
   }
 
+  onSearchEngineChange (e) {
+    const newValue = e.target.value
+    this.settings.selected_search_engine = parseInt(newValue)
+    beaker.browser.setSetting('selected_search_engine', this.settings.selected_search_engine)
+    this.requestUpdate()
+  }
+
+  removeSearchEngine (i) {
+    // decrement selected search engine so it points to the same one if the removed index is less than current one
+    if (i<this.settings.selected_search_engine){
```

Comment:
Sorry to nitpick but can you put spaces around the `<`?

Reply:
No worries - thought I got them all.